src/build/env.py

```diff
@@ -73,7 +73,7 @@ def _symlink_relative(self, path):  # type: (Optional[str]) -> None
         if not path:  # pragma: no cover
             return
 
-        prefix = sysconfig.get_config_var('prefix')
+        prefix = sysconfig.get_config_var('base' if sys.version_info[0] == 2 else 'prefix')
```

Comment:
Could be worth documenting why the discrepency?

Reply:
Oh, it was leftover from my initial tries. I just checked and this is actually not needed. Thanks :blush: